name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/dialogs/transactions/InvestmentPlanModel.java

```diff
@@ -45,6 +45,55 @@ public enum Properties
 
     private IStatus calculationStatus = ValidationStatus.ok();
 
+    public enum Intervals
+    {
+        WEEKLY(100), //
```

Comment:
Great idea. I was always thinking about adding a new property (lots of work).
I propose to use (interval - 100) = week.
This way we leave the space >0 <100 for months, and >100 <200 for weeks.
And have enough "space" for future uses.